db/db_impl.h

```diff
@@ -600,9 +625,14 @@ class DBImpl : public DB {
 
   void InsertRecoveredTransaction(const uint64_t log, const std::string& name,
                                   WriteBatch* batch, SequenceNumber seq,
-                                  size_t batch_cnt) {
-    recovered_transactions_[name] =
-        new RecoveredTransaction(log, name, batch, seq, batch_cnt);
+                                  size_t batch_cnt, bool unprepared) {
```

Comment:
can we rename it to unprepared_batch to make it easier to distinguish from other unprepared variable?